syncd/syncd.cpp

```diff
@@ -864,12 +1243,20 @@ sai_status_t processEvent(swss::ConsumerTable &consumer)
 
     SaiAttributeList list(object_type, values, false);
 
-    if (api != SAI_COMMON_API_GET)
-        translate_vid_to_rid_list(object_type, list.get_attr_count(), list.get_attr_list());
-
     sai_attribute_t *attr_list = list.get_attr_list();
     uint32_t attr_count = list.get_attr_count();
 
+    if (isInitViewMode())
+    {
+        return processEventInInitViewMode(object_type, str_object_id, api, attr_count, attr_list);
+    }
+
+    if (api != SAI_COMMON_API_GET)
+    {
+        // can't be called on init view, since some VID's may have no RID yet
```

Comment:
>// can't be called on init view, since some VID's may have no RID yet [](start = 8, length = 69)

we probably do not need this comment here as init view is handled completely differently.